force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,48 @@
+trigger OpportunityTrigger on Opportunity (before update, before delete) {
+  if (trigger.isBefore && trigger.isUpdate) {
+    for (opportunity opp : Trigger.new) {
+      if (opp.Amount < 5000) {
```

Comment:
```suggestion
      if (opp.Amount <= 5000) {
```
Probably <= given it says the amount must be GREATER than 5000. So 5000 would still be error.